ui/apps/platform/src/Containers/Policies/PatternFly/Wizard/Step3/PolicyCriteriaForm.tsx

```diff
@@ -38,20 +39,18 @@ function PolicyCriteriaForm() {
     }
 
     const isImageSigningEnabled = useFeatureFlagEnabled('ROX_VERIFY_IMAGE_SIGNATURE');
+    const isNetworkPolicyFieldsEnabled = useFeatureFlagEnabled('ROX_NETPOL_FIELDS');
     React.useEffect(() => {
         if (values.eventSource === 'AUDIT_LOG_EVENT') {
             setDescriptor(auditLogDescriptor);
         } else {
-            const descriptors = isImageSigningEnabled
-                ? [
-                      ...policyConfigurationDescriptor,
-                      ...networkDetectionDescriptor,
-                      imageSigningCriteriaDescriptor,
-                  ]
-                : [...policyConfigurationDescriptor, ...networkDetectionDescriptor];
-            setDescriptor(descriptors);
+            setDescriptor([
+                ...policyConfigurationDescriptor,
+                ...networkDetectionDescriptor,
+                ...(isImageSigningEnabled ? [imageSigningCriteriaDescriptor] : []),
+                ...(isNetworkPolicyFieldsEnabled ? networkPolicyFieldDescriptors : []),])
```

Comment:
style-checks might fail with prettier lint error for `])` at end of line to be on its own line with same indentation level as `setDescriptor([`

Reply:
What does `prettier` expect in this case? Something like:
```
setDescriptor([
  ...
])
```